setup/pi/setup-teslausb

```diff
@@ -918,11 +918,13 @@ sed -i '/source \/root\/bin\/setup-teslausb/d' /root/.bashrc
 if ! grep -q TESLAUSB_TIP1 /root/.bashrc
 then
   cat >> /root/.bashrc <<- EOC
+  if [ -n "\$PS1" ]; then
```

Comment:
This line and the closing `fi` below should be tab-indented.
(see `man bash` for how the `<<-` redirection operator works)

Reply:
Ahh, thank you. I replaced the space indents with tabs, and also improved other indentation after reading the docs.